.github/workflows/CodeOwnersSelfMerge.yml

```diff
@@ -1,18 +0,0 @@
-name: Codeowners merging
-on:
-  pull_request_target: { types: [opened] }
-  issue_comment: { types: [created] }
-  pull_request_review: { types: [submitted] }
```

Comment:
So this was running on every PR, not just ones that edit the READMEs?

Reply:
Yes, and every comment